src/plugins/unified_field_list/public/components/field_visualize_button/field_visualize_button.tsx

```diff
@@ -27,7 +27,6 @@ export interface FieldVisualizeButtonProps {
   contextualFields?: string[]; // names of fields which were also selected (like columns in Discover grid)
   trackUiMetric?: (metricType: UiCounterMetricType, eventName: string | string[]) => void;
   buttonProps?: Partial<EuiButtonProps>;
-  wrapInContainer?: (element: React.ReactElement) => React.ReactElement;
```

Comment:
Removing the `wrapInContainer` prop as this button is only ever used inside `FieldPopoverVisualize` which always passes in the wrapper function to wrap this in a eui footer..
So for code simplicity I've removed function and explicitly wrapped the button in a `EuiPopoverFooter`